lib/pbench/server/hierarchy.py

```diff
@@ -344,6 +351,7 @@ def __init__(self, name, path, config):
                 "Invalid unpacking directories (missing tar ball):"
             ),
             self.TARBALL_LINKS: DictOfList("Invalid tar ball links:"),
+            self.IMPOSSIBLE: DictOfList("Impossible Conditions:"),
```

Comment:
For a moment while skimming these changes, my mind slipped out of "programming" mode into "real English" and I suffered the realization that we like to use "impossible" in a really pointless and rather arrogant sense. If it happened, of course it's possible; if it was truly impossible, we wouldn't need this. I suppose we should say "unlikely" or "unexpected" or even "undesired".

Or perhaps... "inconceivable" which would at least give us a famous Princess Bride line for those familiar with the book or movie.